docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,264 @@
+
+   ![Visual Studio Code Marketplace](media/vscode_marketplace.png)
+
+2. In the **Extensions** view's **Search Extensions in Marketplace** field, enter **Tizen Web**.
+3. Locate **Visual Studio Code Extension for Tizen Web** in the search results and click **Install**.
+4. After the installation is complete, activate the extension by clicking **Reload**.
+5. Once the extension has been activated, to use the extension effectively, you need to install the SDK.
+6. Open the **Command Palette** and enter **Create Wizard Project**.
+7. Extension is a convenient way to create a project. it asks whether you want to use an existing Tizen baseline SDK installation or install a new one:
+
+   ![Visual Studio Code Marketplace](media/tizen_web1.png)
+
+   - To perform a fresh installation, click **YES**.
+
+      Click **YES** to begin installing the Tizen baseline SDK. If it is already installed, skip the installation.
+
+     > **Note**  
+     > If you click **NO**, it is judged that the installation has already been made and the project is created.
+
+## Install from a VSIX File
+
+To install VS Code Extension for Tizen Web from a VSIX file:
+
+1. In Visual Studio Code, at the bottom of the **Activity Bar**, click the **Extensions** icon.
+2. In the top right corner of the **Extensions** view, click the **More** button (![More icon](media/vscode_more_icon.png)) and select **Install from VSIX**.
+
+   ![Install from VSIX](media/tizen_web2.png)
+
+3. Select the VSIX file in the file browser and click **Install**.
+4. After the installation is complete, activate the extension by clicking **Reload**.
+
+   ![Install from VSIX](media/tizen_web3.png)
+
+5. Once the extension has been activated, Open the Command Palette and enter Create Wizard Project. It asks whether you want to use an existing Tizen baseline SDK installation or install a new one:
+
+   ![Baseline SDK installation query](media/tizen_web5.png)
+
+   - To perform a fresh installation, click **YES**.
+
+     Click **YES** to begin installing the Tizen baseline SDK. If it is already installed, skip the installation.
+
+     > **Note**  
+     > If you click **NO**, it is judged that the installation has already been made and the project is created.
+
+
+## Install Emulator Images
+
+If you do not have a real device, you can run applications in the Tizen emulator.
+
+To download emulator images, you can use the Tizen Package Manager or the Tizen Emulator Manager.
+
+- To use the Tizen Package Manager:
+  1. Open the **Command Palette** and enter **Run Package-Manager**.
+  2. Select the profiles and versions that you want to install and click **Install**.
+
+     ![Package Manager](media/tizen_package_manager.png)
+
+- To use the Tizen Emulator Manager:
+
+  > **Note**  
+  > The Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
+
+  1. Open the **Command Palette** and enter **Run Emulator-Manager**.
```

Comment:
@sabeena21 
Running "Run Emulator-Manager" has the same result as media/howtoinstall-emulatormanager.png file.

Execute Emulator-Manager because it is a step for Emulator-Manager.

Reply:
done.